app/consumer/app.go

```diff
@@ -334,6 +334,7 @@ func New(
 		&app.IBCKeeper.PortKeeper,
 		app.IBCKeeper.ConnectionKeeper,
 		app.IBCKeeper.ClientKeeper,
+		nil,
```

Comment:
Could we pass in staking keeper here in case we'd ever want to test this upgrade with a normal consumer?

Reply:
Staking keeper will only be used for democracy one, but could pass consumer keeper here since consumer keeper implement the interface of staking keeper. On consumer app, there's no staking keeper FYI.

Reply:
The reason for not passing anything here is to show that it's not democracy app

Reply:
For all the uses of stakingKeeper, it has checker for `stakingKeeper == nil` check FYI